### PR TITLE
Fix(cypress and prettier): fix package.json and cypress.json

### PR DESCRIPTION
**Pull request by siddanand**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
     "coverage:report": "nyc report --check-coverage --statements 50",
     "lint": "next lint",
     "lint:fix": "next lint --fix",
-    "prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --ignore-unknown",
-    "prettier:check": "npm run prettier -l",
-    "prettier:fix": "npm run prettier --write"
+    "prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --write --ignore-unknown",
+    "prettier:check": "npx eslint-config-prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}'",
```
Comment (chirgjn) [inline package.json:18]: As per https://github.com/prettier/eslint-config-prettier#cli-helper-tool this check for conflicts between `prettier` and `eslint` rules. The expectation is to check for formatting errors.

File: package.json [hunk 2]
```diff
@@ -14,9 +14,9 @@
     "coverage:report": "nyc report --check-coverage --statements 50",
     "lint": "next lint",
     "lint:fix": "next lint --fix",
-    "prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --ignore-unknown",
-    "prettier:check": "npm run prettier -l",
-    "prettier:fix": "npm run prettier --write"
+    "prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --write --ignore-unknown",
```
Comment (chirgjn) [inline package.json:17]: This command is meant to just display the prettier errors in the terminal, not fix them. We have `prettier:fix` for that.

**Review by chirgjn — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (siddanand) [reply package.json:17]: Pretter:fix is not working, this command also just shows the error like prettier:check does. It is not writing those changes.

File: package.json (hunk 1 above)
Comment (siddanand) [reply package.json:18]: Okay will change this.

File: package.json (hunk 2 above)
Comment (siddanand) [reply package.json:17]: "prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --ignore-unknown",
    "prettier:check": "npm run prettier -l",
    "prettier:fix": "npm run prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --write --ignore-unknown"

Can I do something like this?

Comment (siddanand) [reply package.json:17]: "prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --ignore-unknown",
"prettier:check": "npm run prettier -l",
"prettier:fix": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --write --ignore-unknown"

Or this, to be more precise

Comment (chirgjn) [reply package.json:17]:
```
"prettier": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --ignore-unknown",
"prettier:check": "npm run prettier -l",
"prettier:fix": "npm run prettier --write"
```

This does not work?

Comment (siddanand) [reply package.json:17]: No, the prettier:fix: npm run prettier --write  doesn't work.

"prettier:fix": "prettier '**pages/*.{js,jsx,vue}' '**pages/api/*.{js,jsx,vue}' --write --ignore-unknown". This does.

Comment (chirgjn) [reply package.json:17]: Let's go with [your suggestion](https://github.com/pesto-students/babitz-frontend/pull/19#discussion_r677605891)

**Review by chirgjn — APPROVED** [review]

**Pull request merged**